internal/netutil/ident.go

```diff
@@ -162,17 +163,17 @@ func uidFromLsof(lip net.IP, lport int, rip net.IP, rport int) (uid int, err err
 		//"-a",  // AND the following together:
 		// "-u", strconv.Itoa(uid)  // just this uid
 		"-itcp") // we only care about TCP connections
+
 	stdout, err := cmd.StdoutPipe()
 	if err != nil {
 		return
 	}
-	defer cmd.Wait()
-	defer stdout.Close()
-	err = cmd.Start()
-	if err != nil {
+	if err = cmd.Start(); err != nil {
 		return
 	}
```

Comment:
this func is too long (more than 2-3 lines) for naked returns. Use `err := cmd.Start()` here and `return err` explicitly so caller doesn't have to reason about the scope

Likewise below.